.github/workflows/ci.yml

```diff
@@ -46,6 +46,20 @@ jobs:
 
       - name: flake8
         run: flake8 .
-
+      
```

Comment:
You should review the diffs when you're done, and things like this shouldn't be introduced by mistake.